content/en/guides/directory-structure/nuxt-config.md

```diff
@@ -98,24 +98,6 @@ See more on the [dev property](/guides/configuration-glossary/configuration-dev
 
 </base-alert>
 
-### env
```

Comment:
At least current version of nuxt didn't deprecate `env` option and it has it's purpose of inlining environment variables (We also auto add `NUXT_ENV_*` vars).

I think it worth to be kept in docs with some explanation about use-case and difference (not sure if any other page exists regarding `env` option or not)

Reply:
ok improved this @pi0 let me know if it is better or you want to add further explantion

Reply:
Seems awesome 👍